### PR TITLE
CONCF-535 Add mobileInContent ad to the Curated Main Page

### PR DESCRIPTION
**Pull request by bognix**
ping @rogatty @gabrys @hakubo

### CODE REVIEW

**Comment by vforge** [comment]
LGTM

File: front/scripts/main/models/ArticleModel.ts [hunk 1]
```diff
@@ -195,6 +195,7 @@ App.ArticleModel.reopenClass({
 			}
 			if (source.mainPageData) {
 				data.mainPageData = source.mainPageData;
+				data.isCuratedMainPage = true;
```
Comment (rogatty) [inline front/scripts/main/models/ArticleModel.ts]: Consult with @vforge if this is the right place to do that. It depends on where we check the Optimizely bucket.

File: front/scripts/main/views/ArticleView.ts [hunk 2]
```diff
@@ -46,9 +46,20 @@ App.ArticleView = Em.View.extend(App.AdsMixin, App.ViewportMixin, {
 
 	articleContentObserver: function (): void {
 		var model = this.get('controller.model'),
-			article = model.get('article');
+			article = model.get('article'),
+			isCuratedMainPage = model.get('isCuratedMainPage');
 
-		if (article && article.length > 0) {
+		if (isCuratedMainPage) {
```
Comment (rogatty) [inline front/scripts/main/views/ArticleView.ts]: Keep all the conditions consistent. If here we check `isCuratedMainPage` then we should check it in the template too. Sync with @vforge

File: front/scripts/main/models/ArticleModel.ts (hunk 1 above)
Comment (bognix) [inline front/scripts/main/models/ArticleModel.ts]: talked offline. @vforge will take care of that

Comment (vforge) [inline front/scripts/main/models/ArticleModel.ts]: Done in https://github.com/Wikia/mercury/commit/bbfff050e3532b2178357ca6452bb8d51f864ef9

File: front/scripts/main/views/ArticleView.ts (hunk 2 above)
Comment (vforge) [inline front/scripts/main/views/ArticleView.ts]: Done in https://github.com/Wikia/mercury/commit/bbfff050e3532b2178357ca6452bb8d51f864ef9

**Comment by rogatty** [comment]
LGTM

**Comment by hakubo** [comment]
LGTM

**Comment by rogatty** [comment]
@gabrys @nandy-andy: wanna take a look?

File: front/scripts/main/mixins/AdsMixin.ts [hunk 3]
```diff
@@ -51,6 +51,26 @@ App.AdsMixin = Em.Mixin.create({
 		}
 	},
 
+	/**
+	 * @desc Load ads for main page.
+	 * InContent ad should be displayed below curated content only when it's available.
+	 * Prefooter ad should be loaded above footer
```
Comment (gabrys) [inline front/scripts/main/mixins/AdsMixin.ts]: Is this comment outdated?

**Comment by gabrys** [comment]
LGTM

File: front/scripts/main/mixins/AdsMixin.ts (hunk 3 above)
Comment (bognix) [inline front/scripts/main/mixins/AdsMixin.ts]: no

**Pull request merged**